ulauncher/search/SortedList.py

```diff
@@ -43,7 +43,15 @@ def extend(self, items):
             self.append(item)
 
     def append(self, result_item):
-        score = get_score(self._query, result_item.get_search_name())
+        # get_search_name() returns a string with the app display name, but it may contain a
+        # second line in which case that line is the name of the executable
+        search_fields = result_item.get_search_name()
+        name, exec_name, *_ = f'{search_fields}\n'.split('\n')
```

Comment:
I can't decide if I think this line is clever or ugly. I'm leaning towards being counted as clever in other languages, but not "Pythonic". The problem is that result_item.get_search_name() returns just the name for SortedLists that are not a applications.

Reply:
I vote clever as well.